tests/data/flatRun.yaml

```diff
@@ -1,6 +1,6 @@
 2021052500077:
-  FAILURES:
-  - RXX_S00 C07 NOISE
+  SUCCESS: false
```

Comment:
I'm not sure I understand this.  `SUCCESS: false` should be followed by `FAILURES: [not-an-empty-list]`.  I'm sure this is what the source file had, and I'll have a look once this ticket merges to see if I can understand this more.

Reply:
Oh huh, good point.  I'm just copying in what it has but I agree it doesn't make sense.